### ISSUE TITLE
Viewing a portugeuse resource causes javascript to not execute and language preference switch

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/coop-library#187]
**Describe the bug**
When viewing the resource "_Uberização do trabalho, com o professor Rodrigo Carelli, MPT/RJ – YouTube_", Javascript does not execute properly on the page as seen in the image below:

![image](https://user-images.githubusercontent.com/564762/74263849-9761d800-4ccd-11ea-901a-68bb7dee74f0.png)

The following error appears in the console:

```
TypeError: this.menu is null pinecone.umd.js:678
    addEventListeners pinecone.umd.js:678
    Menu pinecone.umd.js:557
    init common.js:8
    fire Router.js:38
    loadEvents Router.js:53
    js main.js:28
```

Going to home page then causes an error "Sorry, no results were found". The above error appears in the console again, and the language preference has been changed as seen in this image:

![image](https://user-images.githubusercontent.com/564762/74264029-f889ab80-4ccd-11ea-98a6-d396255e3e2f.png)

**To Reproduce**
1. Open a browser session in Private / Incognito mode (this helps prevent your cookie from being messed up by this bug)
2. View the resource "_Uberização do trabalho, com o professor Rodrigo Carelli, MPT/RJ – YouTube_"
3. Notice how the JavaScript is not running properly (i.e. site navigation is missing, and Share menu is expanded), and there is an error in the console.
4. Now go to the home page by selecting the logo at the top-left corner.
5. Notice the "Sorry, no results were found" error, and the region code (i.e. "mysite.test/pt") in the browser's address bar has changed from your default to Portuguese.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox 72.0.2, Edge-Chromium 79.0.309.71

**Comment by jhung** [comment]
Something else I noticed which may be related to this bug, this particular resource can not be searched. Searching for "_trabalho_" or "_professor_" does not return it in the search results.

**Comment by greatislander** [comment]
@jhung Can you add anything you see in the JS console to this issue?

**Comment by jhung** [comment]
@greatislander - done. An error was being reported. I've updated the issue.

**Comment by greatislander** [comment]
@jhung to prevent the language from switching to Portuguese, disable the 'Detect browser language' setting on the Dashboard &rarr; Polylang &rarr; Settings. The other issue (JavaScript) is addressed in #197.
